flow/size.yaml

```diff
@@ -62,3 +83,107 @@ components:
           type: integer
           default: 1518
           x-field-uid: 2
+    Flow.SizeWeightPairs:
+      type: object
+      description: |-
+        Custom frame size distribution, defined as <size, weight> pairs (including IMIX distribution).
+      properties:
+        choice:
+          type: string
+          default: predefined
+          x-field-uid: 1
+          x-enum:
+            predefined:
+              x-field-uid: 1
+            custom:
+              x-field-uid: 2
+        predefined:
+          type: string
+          default: imix
+          description: |-
+            Option allow to specify a predefined frame size distribution <size, weight> pairs (including IMIX distribution).  
+            The available predefined distribution pairs are:
+            - IMIX (64:7, 570:4, and 1518:1)  
+            - IPSec IMIX (90:58.67, 92:2, 594:23.66 and 1418:15.67)  
+            - IPv6 IMIX (60:58.67, 496:2, 594:23.66 and 1518:15.67)  
+            - Standard IMIX (58:58.67, 62:2, 594:23.66 and 1518:15.67)  
+            - TCP IMIX (90:58.67, 92:2, 594:23.66 and 1518:15.67)  
+            - RPR Trimodal (64:60, 512:20 and 1518:20)  
+            - RPR Quadmodal (64:50, 512:15, 1518:15 and 9216:20)  
+            - Cisco (64:7, 594:4 and 1518:1)  
+            - Tolly (64:55, 78:5, 576:17 and 1518:23)  
+          x-field-uid: 2
+          x-enum:
+            imix:
+              x-field-uid: 1
+            ipsec_imix:
+              x-field-uid: 2
+            ipv6_imix:
+              x-field-uid: 3
+            standard_imix:
+              x-field-uid: 4
+            tcp_imix:
+              x-field-uid: 5
+            rpr_trimodal:
+              x-field-uid: 6
+            rpr_quadmodal:
+              x-field-uid: 7
+            cisco:
+              x-field-uid: 8
+            tolly:
+              x-field-uid: 9
+        custom:
+          type: array
+          items:
+            $ref: '#/components/schemas/Flow.SizeWeightPairs.Custom'
+          x-field-uid: 3
+    Flow.SizeWeightPairs.Custom:
+      type: object
+      description: |-
+        Option allow to set a custom frame size distribution <size, weight> pairs.
+      properties:
+        size:
```

Comment:
This raised a question that if we create a IMIX traffic and we see traffic drops , dont we need to have a way to track flows by packet size to see if packets of certain size are the ones getting dropped when running an IMIX test ? ( Relevant more for tracking PR )

Reply:
We should have a feature to track by frame size.